core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -411,8 +411,8 @@ public void close() {
             quorumState.removeEventListeners(blockChain.getBlockChain(), peerGroup);
             quorumRotationState.removeEventListeners(blockChain.getBlockChain(), peerGroup);
             // reset state of chain sync
-            quorumState.initChainTipSyncComplete = false;
-            quorumRotationState.initChainTipSyncComplete = false;
+            quorumState.close();
+            quorumRotationState.close();
```

Comment:
need to reset more of the sync state than this one value.

Reply:
One reason for "Syncing..." was that it was waiting for a reply to a message sent before the network was completely disconnected.  After starting up, the `wait` flag was still set to true.  While waiting, we don't request again.